webapp/channels/src/components/search_results/messages_or_files_selector.tsx

```diff
@@ -23,11 +29,29 @@ type Props = {
     messagesCounter: string;
     filesCounter: string;
     isFileAttachmentsEnabled: boolean;
+    crossTeamSearchEnabled: boolean;
     onChange: (value: SearchType) => void;
     onFilter: (filter: SearchFilterType) => void;
+    onTeamChange: (teamId: string) => void;
 };
 
 export default function MessagesOrFilesSelector(props: Props): JSX.Element {
+    const teams = useSelector((state: GlobalState) => getMyTeams(state));
```

Comment:
note: eventually we'll have to make a `TeamSelector` component that will do this job.